fix: correct the trigger function to make sense for the given example (#41079)
supabase/supabase: apps/docs/content/guides/realtime/subscribing-to-database-changes.mdx

as $$
begin
  perform realtime.broadcast_changes(

>     'topic:' || coalesce(NEW.id, OLD.id) ::text,       -- topic - the topic to which we're broadcasting where we can use the topic id to build the topic name

Reviewer: ```suggestion
    'topic:' || coalesce(NEW.id, OLD.id) ::text,       -- topic - the topic to which you're broadcasting where you can use the topic id to build the topic name
```

Correcting an old error while we're here.
